### ISSUE TITLE
Functionality of the screen is operable through Tab and arrow kays but there is no such indication, Widget Builder issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#807]
**Describe the bug**
Functionality of the screen is operable through Tab and arrow kays but there is no such indication, Widget Builder issue.

**To Reproduce**

Go to Widget Builder  (http://localhost:9992/cm/app/?view=widgetbuilder&site=test2&mode=readonly&name=index.html&path=%2FSites%2Ftest&pathType=page)
Click create new widget.
Content, Resources and Display tabs are accessible through Tab and Arrow key but not indicated.

**Expected behavior**
if Functionality of the screen is operable through Tab and arrow kays then indication should be there.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
Not reproducible. It got fixed while fixing some other bug.
